### PR TITLE
Remove the cleanup logic that has now cleaned up orphaned pods

### PR DESCRIPTION
**Pull request by consideRatio**
This reverts commit 39e6b2c985540ecfe7203833fdbdbf8b8e9e4c39, the main part of #3192. This was something intented to be reverted once we had let it execute in all hubs - it was just a one off cleanup operation.

### CODE REVIEW

**Review by yuvipanda — APPROVED** [review]
This looks good, but GHG center is in the middle of a demo right now, so let's hold off on merging it until after.

I'm happy to merge this in a few hours once that's over!

**Comment by consideRatio** [comment]
Thanks, your most welcome to merge it!
The PR adding it had a 100% success rate so I think it could work out without any issues to merge this as well.

**Pull request merged**
